wfl/configset.py

```diff
@@ -585,3 +612,20 @@ def __str__(self):
         if self.abcd is not None:
             s += _fmt('ABCD connection', str(self.abcd))
         return s
+
+def _from_dict_str_to_dict_Path(files_dict):
+    """ converts all strings in the dictionary  to Paths"""
+
```

Comment:
Not sure if this is the best way?

Reply:
I'm not sure if it's exactly "better", but it's certainly more compact to do
```
    output_files = {Path(k): Path(v) for k, v in output_files.items() }
```
since `Path(Path)` and `Path(str)` both just result in a `Path`.